assets/styles/app.scss

```diff
@@ -22,4 +27,35 @@ header {
 .user-icon {
     width: 2rem;
     height: 2rem;
+}
+
+// HOME PAGE pour le fichier index.html.twig
+
+.banner-img {
+    width: 100%;
+}
+
+.wrapper {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+    margin: 1em auto; 
+    max-width: 800px; 
+    width: 95%; 
```

Comment:
golden rule: do you really need a width here? I guess a `padding: 1rem` would suit better